### ISSUE TITLE
fix-up repo

### ISSUE THREAD

**Issue by adam-cyclones** [adam-cyclones/unpack#8]

**Comment by nombrekeff** [comment]
Yup, readmes are usually like that, although technical segments should remain as similar as possible, so people that have come previously know precisely where to look for some resource.

I will be happy to help you out with it, and If you have any questions or doubts about something, please contact me. I enjoy doing readmes and such!

**Comment by adam-cyclones** [comment]
Thanks nombrekeff, It was so nice to merge a contribution, what a feeling.

**Comment by nombrekeff** [comment]
It's rather cool, isn't it? 

For me, it's always a pleasure when somebody accepts one of my PRs as well 😄  And of course, helping out is always a positive thing 💯
